README.md

```diff
@@ -161,6 +161,7 @@ The table below describes the metastore tunnel configuration values:
 | `*.metastore-tunnel.known-hosts`                        | No       | Path to a known hosts file. |
 | `*.metastore-tunnel.private-keys`                       | No       | A comma-separated list of paths to any SSH keys required in order to set up the SSH tunnel. |
 | `*.metastore-tunnel.timeout`                            | No       | The SSH session timeout in milliseconds, `0` means no timeout. Default is `60000` milliseconds, i.e. 1 minute. |
+| `*.metastore-tunnel.strict-host-key-checking`           | No       | Whether the SSH tunnel should be created with strict host key checking. Can be set to `yes` or `no`. The default is `yes`. |
```

Comment:
There is some info in the readme saying we do tunnelling and you need to: `Note that all the machines in the tunnel expression must be included in the known_hosts file` that needs an update as this can be an alternative to having the known_hosts file.